git.zsh

```diff
@@ -107,6 +195,50 @@ _fzf_complete_git-unstaged-files_post() {
     echo "${(q)filename}"
 }
 
+_fzf_complete_git-gpg-key() {
+    local fzf_options="$1"
+    shift
+    local global_key=$(git config --global user.signingkey)
+    local local_key=$(git config --local user.signingkey)
+    _fzf_complete "--ansi $fzf_options" "$@" < <(
+        [[ -n "$gpg_command" ]] && LANG=C "$gpg_command" --list-secret-keys --keyid-format LONG 2> /dev/null |
+        awk -v global_key="$global_key" -v local_key=$local_key -v prefix="$prefix_option" '
+            /^sec/ {
+                for (i = 1; ;) {
+                    if ($0 == "") {
+                        break
+                    }
+                    if ($1 == "uid") {
+                        uid = $3 " " $4
+                    }
+                    if ($1 == "sec" || $1 == "ssb") {
+                        sub(/.*\//, "", $2)
+                        keys[i] = $2
+                        types[i++] = $4
+                    }
+                    getline
+                }
+                for (j = 1; j < i; ++j) {
+                    message = ""
+                    if (keys[j] == global_key) {
+                        message = message " " "[Git Global]"
+                    }
+                    if (keys[j] == local_key) {
+                        message = message " " "[Git Local]"
+                    }
+                    print prefix keys[j], types[j], uid message
+                }
+                delete keys
+                delete types
+            }
+        ' | _fzf_complete_git_tabularize
+    )
+}
```

Comment:
This implementation has weird behaviour in my environment:

```zsh
$ git commit -S **
>
  4/4
> 0000000000000000  [SC] unknown] Chitoku
  0000000000000000  [E] unknown] Chitoku
  0000000000000000  [SC] Chitoku <odango@chitoku.jp>
  0000000000000000  [E] Chitoku <odango@chitoku.jp>
```

```zsh
$ LANG=C gpg2 --list-secret-keys --keyid-format LONG 2> /dev/null
/home/chitoku/.gnupg/pubring.kbx
--------------------------------
sec   rsa4096/0000000000000000 2016-05-17 [SC]
      0000000000000000000000000000000000000000
uid                 [ unknown] Chitoku <odango@chitoku.jp>
ssb   rsa4096/0000000000000000 2016-05-17 [E] [expires: 2020-05-17]

sec   rsa4096/0000000000000000 2020-01-12 [SC]
      0000000000000000000000000000000000000000
uid                 [ultimate] Chitoku <odango@chitoku.jp>
ssb   rsa4096/0000000000000000 2020-01-12 [E]
```

Can you dig into what happens?